docs/README.md

```diff
@@ -525,20 +525,21 @@ referencing as value env vars defined early in the file (like Travis does).
 
 ### free-hosted-runner-disk-space
 
-Removes unnecessary folders and files from a GHA [hosted runner](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners). This action might be useful when we run jobs which require a lot of disk space.
+Removes unnecessary folders and files from a GHA [hosted runner](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners), and if that is not enough, allows to manipulate and utilize unused memory partitions. This action might be useful when we run jobs which require a lot of disk space.
```

Comment:
We didn't add any documentation related to the new behaviour, at the least we should include the possibility to turn on the new behaviour via the newly implemented flag, passing through options to the underlying custom action and linking to the open source repository of that action so that people can look into it with ease.

Reply:
Added a more in-depth description with an example below